### PR TITLE
Installation guide for Google Cloud Platform

### PR DESCRIPTION
**Pull request by keesvanbemmel**
Adds an installation guide for Google Cloud Platform.

One minor detail in this guide needs https://github.com/directus/directus/pull/10911 to be merged.

@rijkvanzanten @benhaynes hope this is what you had in mind. Let me know if I can add more! Furthermore there is a reference to a public repository of mine that has a one-click deployment boilerplate for Directus. This is needed for the `Run on Google Cloud` button. The repo can be found here: https://github.com/keesvanbemmel/directus-docker I wouldn't mind putting this in the directus repo somewhere if that's better. Up to you!

CleanShot 2022-01-09 at 17 55 03@2x

### CODE REVIEW

**Comment by benhaynes** [comment]
Thank you!!! 💯

File: docs/getting-started/installation/gcp.md
Comment (azrikahar) [inline docs/getting-started/installation/gcp.md:85]:
```suggestion
   `https://console.cloud.google.com/billing/linkedaccount?project=<your-project-id>`. Since the database and storage
```

File: docs/getting-started/installation/gcp.md
Comment (azrikahar) [inline docs/getting-started/installation/gcp.md:217]:
```suggestion
- You could setup [caching](/configuration/config-options/#cache) using
```

File: docs/getting-started/installation/gcp.md
Comment (azrikahar) [inline docs/getting-started/installation/gcp.md:220]:
```suggestion
  [Schedule hooks](/extensions/hooks/#schedule) since there is no container to handle those
```

File: docs/getting-started/installation/gcp.md
Comment (azrikahar) [inline docs/getting-started/installation/gcp.md:221]:
```suggestion
  schedules / cron jobs. Again you have two options: Set the minimum number of instances to 1, this will definitely
```

**Review by azrikahar — CHANGES_REQUESTED** [review]
Thanks for the detailed guide! 👍 Just some minor localhost links & typos to update.

**Review by azrikahar — APPROVED** [review]

**Comment by keesvanbemmel** [comment]
Thanks @azrikahar 

Just to be sure, this documentation also references functionality from https://github.com/directus/directus/pull/10911 so that needs to be merged as well (or removed from these docs)

**Comment by keesvanbemmel** [comment]
Forgot to ask @azrikahar do you want me to add the contents of this

**Comment by azrikahar** [comment]
>Forgot to ask do you want me to add the contents of this 

by that do you perhaps mean table of contents? Not quite sure which contents are referred here 😅

**Comment by benhaynes** [comment]
I think @keesvanbemmel is talking about adding the Env Vars to the docs... which would indeed be really helpful. 👍

**Comment by keesvanbemmel** [comment]
No I pressed send before completing my comment. Edited original comment!

**Comment by rijkvanzanten** [comment]
> do you want me to add the contents of my own public repo to a directus repo?

Yeah lets do that! We can put it on the directus-community org 👍🏻 

Looking at it, it seems to be an "empty" installation. Do we even need to have a repo for that? 🤔

**Comment by rijkvanzanten** [comment]
Makes sense! Yeah lets move that repo into directus-community then. Wondering if we can make it a more generic repo, as the setup is "plain" (not really google cloud specific 🤔).

**Comment by keesvanbemmel** [comment]
> Makes sense! Yeah lets move that repo into directus-community then. Wondering if we can make it a more generic repo, as the setup is "plain" (not really google cloud specific 🤔).

Sure! How?

**Comment by rijkvanzanten** [comment]
@keesvanbemmel I've setup https://github.com/directus-community/gcp-example and sent you an invite for maintainer access 🙂 

Lets put the example code there, and then update this doc to reference that

**Comment by keesvanbemmel** [comment]
Awesome, will do so tomorrow!

**Comment by keesvanbemmel** [comment]
Yes, example repo moved/copied to directus-community and ammended documentation. Also added the Google Cloud deploy button to the readme next to the others.

WOOOP!

**Comment by benhaynes** [comment]
I'll leave it up to @rijkvanzanten to confirm... but wanted to say thank you for this!! 💯

**Review by rijkvanzanten — APPROVED** [review]
Looks good to me! Thanks @keesvanbemmel ❤️

**Pull request merged**
